extensions/kafka/src/main/java/com/hazelcast/jet/kafka/KafkaDataConnection.java

```diff
@@ -100,6 +102,21 @@ public <K, V> Consumer<K, V> newConsumer() {
         return new KafkaConsumer<>(getConfig().getProperties());
     }
 
+    /**
+     * Creates new instance of {@link KafkaConsumer} based on the DataConnection
+     * configuration.
```

Comment:
This should say something like 
```suggestion
     * Creates new instance of {@link KafkaConsumer} based on the DataConnection
     * configuration and given properties parameter.
```